word_game.rb

```diff
@@ -0,0 +1,160 @@
+require 'pry'
+require 'faker'
+
+class Game
+  def initialize
+    @word = GameData.new.easy_word.downcase
+    @user_choice = ''
+    @bad_array = []
+    @guess_string = ''
+    @change_split_word = []
+    @picture = GameData.new.picture
+  end
+  def level
+    print "would you like to play easy or hard?\n"
+    answer = gets.chomp.downcase
+    if answer == "easy"
+      print "I hope you know your Pokemon.\n"
+      @word = GameData.new.easy_word.downcase
+    else
+      print "There are no spaces in this Rick and Morty quote.\n"
+      @word = GameData.new.hard_word.downcase
+    end
+  end
+
+
+  def summary
+    puts "This is a summary the word is #{@word}"
+    puts "This is a check on #{@picture}"
+    puts "This is a check on #{@word}"
+  end
+
+  # Verbs
+  # input
+  def user_input
+    puts 'Choose a letter please'
+    @user_choice = gets.chomp.downcase
```

Comment:
I don't know that I agree with the decision to make `@user_choice` an instance variable. This value is transient, since it will change whenever the user makes another guess, and this is usually a good indicator that an instance variable is not the right choice.

Instead, you could store the letter as a local variable, and pass it to `guess` as a parameter.